third_party/move/move-vm/runtime/src/storage/struct_name_index_map.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use move_vm_types::loaded_data::runtime_types::{StructIdentifier, StructNameIndex};
+use parking_lot::{MappedRwLockReadGuard, RwLock, RwLockReadGuard};
+use std::collections::BTreeMap;
+
+#[derive(Clone)]
+struct IndexMap<T: Clone + Ord> {
```

Comment:
I thought we had tests here... I know you are copying the code, but could you please add basic unit tests for struct_name_to_idx and idx_to_struct_name calls?

Reply:
done